### PR TITLE
Groovy Updates for gazebo simulation

### PR DESCRIPTION
**Pull request by fmessmer**

### CODE REVIEW

File: cob_hardware_config/raw3-1/urdf/raw3-1.urdf.xacro [hunk 1]
```diff
@@ -24,30 +24,36 @@
...
+		<parent link="base_link" />
+		<child link="arm_base_link" />
+	</joint>
+  </xacro:macro>
 
-  <xacro:ur10_robot name="arm" parent="base_link">
-      <origin xyz="${arm_x} ${arm_y} ${arm_z}" rpy="${arm_roll} ${arm_pitch} ${arm_yaw}" />
-  </xacro:ur10_robot>
+ <xacro:ur_arm_base_joint/>
+ <xacro:ur10_robot prefix="arm_"/>
```
Comment (abubeck) [inline cob_hardware_config/raw3-1/urdf/raw3-1.urdf.xacro:40]: UR10 has no calibrated origin on raw31?
Please also cleanup the  comments in this file. (They are probably created by myself ;))

File: cob_bringup/robots/raw3-1.launch [hunk 2]
```diff
@@ -10,8 +10,8 @@
 	</include>
 	
 	<!-- upload default configuration parameters -->
-	<include file="$(arg pkg_env_config)/upload_navigation_goals.launch">
```
Comment (abubeck) [inline cob_bringup/robots/raw3-1.launch:13]: Where are the navigation goals coming from now?

File: cob_hardware_config/raw3-1/urdf/raw3-1.urdf.xacro (hunk 1 above)
Comment (fmessmer) [inline cob_hardware_config/raw3-1/urdf/raw3-1.urdf.xacro:40]: This raw3-1.urdf.xacro includes the GeorgiaTech ur10_description.
This description does not have a origin parameter for the macro.
MDL patched it locally.
However, there is a discussion ongoing on the mailinglist whether to make the GeorgiaTech description the default description and push it to ROS-Industrials universal_robot stack.

File: cob_bringup/robots/raw3-1.launch (hunk 2 above)
Comment (fmessmer) [inline cob_bringup/robots/raw3-1.launch:13]: it was commented as the filename was wrong ;-)

**Pull request merged**